thoth/storages/graph/postgres.py

```diff
@@ -3846,26 +3847,35 @@ def create_python_cve_record(
         version_range: str,
         advisory: str,
         cve: str = None,
-    ) -> Tuple[CVE, bool]:
```

Comment:
@fridex at looked at https://github.com/thoth-station/cve-update-job/blob/master/app.py#L87 and it looks for me that we don't need to return CVE at all because we are not using it in `cve-update-job`. Rather, what we are interested is whether we have that cve already in the database that's why WDTY if we return True only if the record exists, otherwise False and we going to log about it accordingly. PTAL at https://github.com/thoth-station/cve-update-job/pull/251/?

Reply:
The approach sounds reasonable :+1: